.gitlab-ci.yml

```diff
@@ -4,7 +4,7 @@ include:
 # SETUP
 
 variables:
-  CURRENT_CI_IMAGE: "10"
+  CURRENT_CI_IMAGE: "12"
```

Comment:
is this new image based on the API 35 one ? I thing you should rebase this PR on top of my existing one otherwise we're going to miss the API 35 image requirements

Reply:
I will rebase it later once your PR is merged, that is why I jumped directly to tag 12.